cmd/bee/cmd/cmd.go

```diff
@@ -238,6 +239,7 @@ func (c *command) setAllFlags(cmd *cobra.Command) {
 	cmd.Flags().String(optionNamePostageContractAddress, "", "postage stamp contract address")
 	cmd.Flags().String(optionNamePriceOracleAddress, "", "price oracle contract address")
 	cmd.Flags().String(optionNameTransactionHash, "", "proof-of-identity transaction hash")
+	cmd.Flags().String(optionNameBlockHash, "", "block hash of the block whose parent is the block that contains the transaction hash")
```

Comment:
Why did you chose to include this as a separate parameter? The blockhash should be the blockhash from the block in which the `optionNameTransactionHash` was included in the chain (or the swap deployment transaction if the tx-hash is not passed in)

Reply:
Standalone nodes have no way to fetch the hash since they do not connect to the blockchain backend.

Reply:
Wouldn't it be better for standalone nodes to just use a default value for txHash and blockhash, so they don't need to deal with this and can just ignore it?